[Serve] JSON logging doc (#37472)
ray-project/ray: doc/source/serve/monitoring.md

For a detailed overview of logging in Ray, see [Ray Logging](configure-logging).

### JSON logging format

> You can enable JSON logging format in serve log file by setting the environment variable `RAY_SERVE_ENABLE_JSON_LOGGING=1`. After setting this environment variable, the logs will be in the following format:

Reviewer: ```suggestion
You can enable JSON formatted logging in the Serve log file by setting the environment variable `RAY_SERVE_ENABLE_JSON_LOGGING=1`. After setting this environment variable, the logs have the following format:
```

Reviewer 2: `JSON-formatted` :)

> You can enable JSON logging format in serve log file by setting the environment variable `RAY_SERVE_ENABLE_JSON_LOGGING=1`. After setting this environment variable, the logs will be in the following format:

Reviewer 3: Does this apply to all logs or just the HTTP Proxy logs? The example implies that it's just HTTP Proxy logs.

Author: this example is from deployment log not from http proxy. And yes it will enable all serve log files.
